### PR TITLE
Prepare for 0.1.0: clean up build, add tests, add array support.

### PR DESCRIPTION
**Pull request by rdblue**

### CODE REVIEW

**Comment by rdblue** [comment]
@rgbkrk, could you take a look?

File: src/main/java/jupyter/MIMETypes.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package jupyter;
+
+public class MIMETypes {
+  public static final String TEXT = "text/plain";
+  public static final String HTML = "text/html";
+}
```
Comment (rgbkrk) [inline src/main/java/jupyter/MIMETypes.java:22]: 👍

Comment (rgbkrk) [reply src/main/java/jupyter/MIMETypes.java:22]: Side note -- custom mimetypes have been starting to be placed [in the jupyter docs](http://jupyter.readthedocs.io/en/latest/reference/mimetype.html#listing-of-custom-mimetypes-used-for-display).

**Review by rgbkrk — APPROVED** [review]
Thank you!

**Comment by rgbkrk** [comment]
Want me to enable Travis for this repository?

**Pull request merged**